### PR TITLE
Task 3: Use mongoose instead of mongodb driver

### PR DESCRIPTION
**Pull request by frostbite127**
Please pull these awesome changes in!

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -18,39 +18,39 @@
 // Task 2 : Connecting with mongo db data base
```
Comment (frostbite127) [inline server.js:18]: Uncomment the line to import the 'http' module as it is currently necessary for creating the server

File: server.js [hunk 2]
```diff
@@ -18,39 +18,39 @@
...
```
Comment (frostbite127) [inline server.js:19]: Uncomment the line to import the 'MongoClient' from 'mongodb' module as it is required for connecting to MongoDB

File: server.js [hunk 3]
```diff
@@ -18,39 +18,39 @@
...
-    // Connect to MongoDB
-    await client.connect();
-    console.log('Connected to MongoDB');
+// const mongoConnectionUri = "mongodb+srv://heyshubham24:Shubham2409@cluster0.cwjosgd.mongodb.net/Slambook?retryWrites=true&w=majority";
+// const client = new MongoClient(mongoConnectionUri);
 
-    // Create HTTP Server
-    const server = http.createServer((req, res) => {
-      res.statusCode = 200;
-      res.setHeader('Content-Type', 'text/plain');
```
Comment (frostbite127) [inline server.js:39]: The function startServer() is not being called after setting up the MongoDB connection and creating the HTTP server. Make sure to call startServer() function to run the server

Comment (ShubhamGupta24) [reply server.js:39]: Everything is working fine but uri already given in the task 3 the error occured ,i.e., : MongoServerError: bad auth : Authentication failed.

**Pull request merged**